<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QProgressBar" name="r1_SP">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>100</width>
     <height>750</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> bar-&gt;setStyleSheet(&quot;::chunk {&quot;
                    &quot;background-color: &quot;
                    &quot;qlineargradient(x0: 0, x2: 1, &quot;
                    &quot;stop: 0 green, stop: 0.6 green, &quot;
                    &quot;stop: 0.8 orange, &quot;
                    &quot;stop: 1 red&quot;
                    &quot;)}&quot;);</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="r2_SP">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>250</y>
     <width>100</width>
     <height>750</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="game_time">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>220</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>70</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 75 70pt &quot;MV Boli&quot;;
border-width: 10px;
border-style: solid;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="r1_HP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>550</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QProgressBar::chunk {
     background-color: red;
     width: 20px;
 }</string>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%p</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="r2_HP">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>30</y>
     <width>550</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"> QProgressBar::chunk {
     background-color: red;
     width: 20px;
 }</string>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%p</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1p">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>220</width>
     <height>270</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_1p{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2p">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>148</y>
     <width>220</width>
     <height>270</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2p{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="r1_win_count">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>40</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>25</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 75 25pt &quot;MV Boli&quot;;
border-width: 5px;
border-style: solid;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="r2_win_count">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>20</y>
     <width>40</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>25</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 75 25pt &quot;MV Boli&quot;;
border-width: 5px;
border-style: solid;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_restart">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>300</y>
     <width>550</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_restart{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
